### ISSUE TITLE
baseline 

### ISSUE THREAD

**Issue by Ellohiye** [SHI-Labs/FcF-Inpainting#12]
Hello author, when you compared other baseline models, did you use the mask generated in this paper and the corresponding data set to retrain the baseline model? Looking forward to your reply！

**Comment by praeclarumjj3** [comment]
Hi @Ellohiye, thanks for your interest in our work. As mentioned in our paper, we only train an equivalent PyTorch re-implementation of CoModGAN for comparisons at 256x256 resolution.

For all other baseline comparisons (including CoModGAN AT 512x512 resolution), we use their publicly released models.

**Comment by Ellohiye** [comment]
> 

Thank you very much for your detailed reply! I want to do a comparative experiment recently, can I ask you some questions? For each model (other baseline models), do I have to guarantee that the mask is the same for all models when training? For example, I trained my model according to the design of the 512*512 image mask in lama. When I compare AOTGAN, can I directly use the official pre-trained model to evaluate the results, or must I retrain AOTGAN on the same mask?

**Comment by praeclarumjj3** [comment]
Hi @Ellohiye, it depends on what factors you are considering while comparing. Most inpainting frameworks also include the type of masks used during training as a part of their proposed method. So, ideally, for a table in a research article. you should use the official model.

**Comment by Ellohiye** [comment]
> Hi @Ellohiye, it depends on what factors you are considering while comparing. Most inpainting frameworks also include the type of masks used during training as a part of their proposed method. So, ideally, for a table in a research article. you should use the official model.

Sorry for the late reply, thank you for your patience, it helped me a lot! Thank you and wish you a happy life!
